GTFS.DB.PostgreSQL/Collections/PostgreSQLAgencyCollection.cs

```diff
@@ -166,6 +178,36 @@ public IEnumerable<Agency> Get()
             });
         }
 
+        /// <summary>
+        /// Returns entity ids
+        /// </summary>
+        /// <returns></returns>
+        public IEnumerable<string> GetIds()
+        {
+            #if DEBUG
+            var stopwatch = new System.Diagnostics.Stopwatch();
+            stopwatch.Start();
+            Console.Write($"Fetching agency ids...");
+            #endif
+            var outList = new List<string>();
+            using (var command = _connection.CreateCommand())
+            {
+                command.CommandText = "SELECT DISTINCT(id) FROM agency";
```

Comment:
Are you sure this should be distinct? There should never be duplicates in here so if the same id is coming back twice you probably want to know about it. Just a thought.

Reply:
bloody copy and paste

Reply:
I suspect there will be many many more

Reply:
fixed